finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -21,11 +21,18 @@ import scala.io.Source
  */
 private object ClassPath {
 
-  private val ignoredPackages = Seq(
+  private[util] var ignoredPackages = Seq(
     "apple/", "ch/epfl/", "com/apple/", "com/oracle/",
     "com/sun/", "java/", "javax/", "scala/", "sun/", "sunw/")
 
   /**
+   * Adding additional packages to be ignored.
+   */
+  def addIgnoredPackages(packages: String*) = {
```

Comment:
seems like you wanted to expose this for testing, but we could instead make ignoredPackages a method, or else add a private[util] method version of ignoredPackages.